assets/app/view/game/discard_trains.rb

```diff
@@ -7,6 +7,10 @@ module Game
     class DiscardTrains < Snabberb::Component
       include Actionable
 
+      def current_entity_actions
+        @current_entity_actions ||= @game.round.actions_for(@game.round.active_step&.current_entity) || []
+      end
```

Comment:
There is only one method (render). The call can go directly into render.

Reply:
I've inlined the method